### PR TITLE
Fixes part of #2314: Changes in Numeric Input Interaction Item[a11y]

### PR DESCRIPTION
**Pull request by Arjupta**
## Explanation

Fixes part of #2314 
Tried setting 1dp stroke of blue color instead of blue shadow. This solved the contrast issue but deformed the Text Input Layout.

|Before|After|
|:-:|:-:|
|Screenshot_1611422697|Screenshot_1611422466|

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Arjupta** [comment]
@rt4914 I think it is because of the change in the space occupied by background drawable. I tried setting the `android:gravity="center_vertical"` for numeric_input_interaction view but it still no effect

**Comment by rt4914** [comment]
Will check this on Monday. Thanks.

**Comment by rt4914** [comment]
The `XX_interaction_item` contains `paddingBottom` just remove that and it should work.
Note: XX means `fraction`, `numeric_input`, `text_input`, or anything in general which uses this file.

Also, because of these changes the `shadow` effect is almost completely gone and looks like a simple border so will need to improve on that part. Thanks.

**Comment by Arjupta** [comment]
> The `XX_interaction_item` contains `paddingBottom` just remove that and it should work.
> Note: XX means `fraction`, `numeric_input`, `text_input`, or anything in general which uses this file.
> 
> Also, because of these changes the `shadow` effect is almost completely gone and looks like a simple border so will need to improve on that part. Thanks.

Yes it did worked out. This is the new UI after removing bottom padding and blue shadow. Scans are also passed for it. 

|After Changes|Tests|
|:-:|:-:|
|Screenshot_1611662464|Screenshot_1611662494|

@rt4914 PTAL

**Comment by rt4914** [comment]
@mschanteltc Will this new UI work where the input box does not have a shadow but just a border?

**Comment by mschanteltc** [comment]
> @mschanteltc Will this new UI work where the input box does not have a shadow but just a border?

I would still prefer it to have the shadow when the user taps on the input. According to this "Text input focus style" [example](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html), even if the shadow does not provide sufficient contrast, the cursor's contrast with the white background should be enough of an indicator to show that the text input is in focus. 

Would it be possible to ignore the contrast rules for this case?

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ARJUPTA Try this: https://gist.github.com/rt4914/19e8f5005c8ee26933aa0e17cc44c248

**Comment by Arjupta** [comment]
|After Suggested change| Tests |
|:-:|:-:|
|Screenshot_1611852226|Screenshot_1611852235

@mschanteltc this is after what @rt4914 suggested, the scans pass for this too and also the UI isn't much changed. Can we go with this?

**Comment by rt4914** [comment]
@ARJUPTA Can you please update the code as per https://github.com/oppia/oppia-android/pull/2548#pullrequestreview-578514487

**Comment by Arjupta** [comment]
> @ARJUPTA Can you please update the code as per [#2548 (review)](https://github.com/oppia/oppia-android/pull/2548#pullrequestreview-578514487)

Done

**Comment by mschanteltc** [comment]
> @mschanteltc this is after what @rt4914 suggested, the scans pass for this too and also the UI isn't much changed. Can we go with this?

Yes this looks great! Let's go with this.

File: app/src/main/res/drawable/edit_text_background_shadow.xml [hunk 1]
```diff
@@ -92,6 +92,9 @@
     <shape>
       <corners android:radius="4dp" />
       <solid android:color="@color/white" />
+      <stroke
+        android:width="1dp"
+        android:color="@color/blue_shade" />
     </shape>
   </item>
-</layer-list>
+</layer-list>
```
Comment (rt4914) [inline app/src/main/res/drawable/edit_text_background_shadow.xml]: EOF is missing.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ARJUPTA One nit change.

File: app/src/main/res/drawable/edit_text_background_shadow.xml (hunk 1 above)
Comment (Arjupta) [reply app/src/main/res/drawable/edit_text_background_shadow.xml]: done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**